packages/kanso-app-settings/build/compile.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 function get_doc(ddoc_url, settings, callback) {
     couchr.get(ddoc_url, function(err, resp){
         if (err && err.error === 'unauthorized') return auth(ddoc_url, settings, callback);
-        if (err && err.response.statusCode === 404) return callback(null, {}); // no design doc. First push.
+        if (err && err.response && err.response.statusCode === 404) return callback(null, {}); // no design doc. First push.
```

Comment:
Not really related, but it is a random bug in this kanso stuff that occurs if CouchDB isn't running